src/client/reducers/store.js

```diff
@@ -0,0 +1,14 @@
+import { ADD_YEAR } from "actions/store";
```

Comment:
Looks good.
I think we should define the datas that we want persistent for the client, to implement the retrieving of thoses datas on the store maybe?
Something like 
[Datas that we would like to persist forever (db)] -> login/config/theme
[Datas that we would like to persist for the session (cache/server)] -> choosen name/socketId?/lobby joined
[Datas that we don't ned to persist at all] -> Board/joined game?

Reply:
Yes all data will be there.
As we're not using Redux, we have the possibility to define multiple layers of Context.
But we need to discuss about it first

Reply:
👌